sematic/db/metrics_queries.py

```diff
@@ -0,0 +1,26 @@
+RUN_COUNT_BY_STATE = """
```

Comment:
Isn't it possible to use the Python API to specify these queries?

Reply:
This one is possible to implement via the Python API. But the one below is likely not because of `strftime('%s', runs.resolved_at)`.